crates/cubecl-linalg/src/matmul/kernels/matmul/algorithm/simple_tma.rs

```diff
@@ -0,0 +1,40 @@
+use super::base;
+use cubecl_core::prelude::*;
+use std::marker::PhantomData;
```

Comment:
can you use core instead of std?

Reply:
Thanks for pointing these out, it's annoying that rust-analyzer has no option to do this by default...

Reply:
I know it is annoying.